site/src/pages/components/selectmenu.mdx

```diff
@@ -188,7 +207,7 @@ You can replace the default listbox part in a similar fashion:
 </selectmenu>
 ```
 
-The `popup` attribute used in this example's `<div popup>` is from the proposal at [Popup API (Explainer)](https://open-ui.org/components/popup.research.explainer). The element with `behavior="listbox"` is required to be a `<div popup=popup>`. Applying `behavior="listbox"` tells the browser to apply select menu listbox behavior to that element: it will be opened when the `<selectmenu>`'s button is clicked, and the user can select `<option>`s inside it with mouse, arrow keys, and touch.
+The `popover` attribute used in this example's `<div popover>` is from the proposal at [Popover API (Explainer)](https://open-ui.org/components/popover.research.explainer). The element with `behavior="listbox"` is required to be a `<div popover="auto">`. Applying `behavior="listbox"` tells the browser to apply select menu listbox behavior to that element: it will be opened when the `<selectmenu>`'s button is clicked, and the user can select `<option>`s inside it with mouse, arrow keys, and touch.
```

Comment:
would it make sense to, instead of the explainer, refer to the Popover API and link to to https://html.spec.whatwg.org/dev/popover.html#the-popover-attribute instead here?